src/compute-model-statistics/src/main/scala/ComputeModelStatistics.scala

```diff
@@ -113,11 +109,50 @@ class ComputeModelStatistics(override val uid: String) extends Transformer with
   /** @group setParam */
   def setEvaluationMetric(value: String): this.type = set(evaluationMetric, value)
 
-  lazy val logger = Logger.getLogger(this.getClass.getName)
+  // Parameter required only is using native Spark ML Estimator instead of MMLSpark Estimator
+  val modelName: Param[String] =
+    StringParam(this, "modelName", "Model name, only required if using SparkML estimators", "SparkML")
+
+  def getModelName: String = $(modelName)
+
+  def setModelName(value: String): this.type = set(modelName, value)
+
+  /** Parameter required only is using native Spark ML Estimator instead of MMLSpark Estimator
+    * Set to true if computing metrics for the output of a classification model, set to false if regression model.
+    */
+  val classifier: Param[Boolean] =
+    BooleanParam(this, "classifier", "Is classifier scored data, only required if using SparkML estimators", true)
```

Comment:
This would not be needed if the above parameter was a pipeline stage because you could case match

Reply:
agreed, it is brittle, but it's the only way to make the case work when bringing in scored data outside of spark

Reply:
I've removed the param now